Strata/Languages/Boogie/StatementSemantics.lean

```diff
@@ -286,3 +288,2106 @@ abbrev EvalStatementContract (π : String → Option Procedure) : BoogieEval →
 abbrev EvalStatementsContract (π : String → Option Procedure) : BoogieEval → BoogieEvalBool →
     BoogieStore → BoogieStore → List Statement → BoogieStore → Prop :=
   Imperative.EvalStmts Expression Command (EvalCommandContract π)
+
+/-! ## Theorems -/
```

Comment:
Could we move these theorems to a separate file? Maybe `StatementSemanticsProps.lean`? Having proofs in separate files makes it easier to wrap your head around the codebase, because you can often skip looking at those files entirely (unless you're trying to prove something, yourself).